clients/itwin/src/Token.ts

```diff
@@ -46,151 +51,7 @@ class TokenPrefixToTypeContainer {
-      if (tokenStr.startsWith(key)) {
-        return new TokenPrefixToTypeContainer.tokenPrefixToConstructorDict[key]();
-      }
-    }
-    throw new BentleyError(AuthStatus.Error, "Invalid access token", Logger.logError, loggerCategory, () => ({ tokenStr }));
-  }
-
-  /**
-   * Creates a strongly typed AccessToken object from an untyped JSON with the same properties as [[AccessToken]]
-   * @param jsonObj
-   * @throws [BentleyError]($bentley) if the supplied tokenResponse is undefined, or does not contain an "access_token" field
-   */
-  public static fromJson(jsonObj: AccessTokenProps): AccessToken {
-    const jwt = jsonObj.tokenString;
-    const startsAt = jsonObj.startsAt !== undefined ? new Date(jsonObj.startsAt) : undefined;
-    const expiresAt = jsonObj.expiresAt !== undefined ? new Date(jsonObj.expiresAt) : undefined;
-    const userInfo = jsonObj.userInfo !== undefined ? UserInfo.fromJson(jsonObj.userInfo) : undefined;
-    return new AccessToken(jwt, startsAt, expiresAt, userInfo);
-  }
-
-  public toJSON(): AccessTokenProps {
-    return {
-      tokenString: this._tokenString,
-      startsAt: this._startsAt?.toJSON(),
-      expiresAt: this._expiresAt?.toJSON(),
-      userInfo: this._userInfo,
-    };
-  }
-
-  /**
-   * Creates an AccessToken from the typical token responses obtained from Authorization servers
-   * - The fields from the token response to different names in AccessToken to keep with naming guidelines
-   * - Only basic validation is done - the input tokenResponse must be defined, and have an "access_token" field in it
-   * @param tokenResponse Response containing the token string as obtained from the authorization server
-   * @param userProfileResponse Response containing the user profile information as obtained from the authorization server
-   * @returns Strongly typed AccessToken
-   * @see [[AccessToken.fromJson]] for use cases that involve serialization/deserialization of AccessToken
-   * @throws [BentleyError]($bentley) if the supplied tokenResponse does not contain an "access_token" field
-   * @internal
-   */
-  public static fromTokenResponseJson(tokenResponse: any, userProfileResponse?: any): AccessToken {
-    if (!tokenResponse || !tokenResponse.access_token)
-      throw new BentleyError(AuthStatus.Error, "Expected tokenResponse to contain access_token field", Logger.logError, loggerCategory, () => tokenResponse);
-
-    const startsAt = new Date((tokenResponse.expires_at - tokenResponse.expires_in) * 1000);
-    const expiresAt = new Date(tokenResponse.expires_at * 1000);
-    const userInfo = userProfileResponse ? UserInfo.fromTokenResponseJson(userProfileResponse) : undefined;
-
-    const token = new AccessToken(tokenResponse.access_token);
-    token._startsAt = startsAt;
-    token._expiresAt = expiresAt;
-    token._userInfo = userInfo;
-    return token;
-  }
-}
+export type AccessTokenString = string;

```

Comment:
As I'm looking at the code, wondering if we should just call this `AccessToken` and use that to mean just the string type rather than having the more verbose `AccessTokenString`.

It'd be a bit confusing at first for anyone who is used to the old type name but I doubt anyone new would be surprised by it. Especially the way it's handled in all other implementations that I've seen.

Reply:
This is probably a good idea. Would probably be confusing for people used to AccessToken being an object, but makes more sense in the grand scheme of things since an accessToken normally just refers to a string anyways.